### PR TITLE
PostgreSQL constraints

### LINKED ISSUE #4646: [Concept Entry] Postgresql constraints
### Reason/inspiration (optional)

We would like a new entry on the `Constraints` concept under PostgreSQL. The entry should go in a new file under `docs/content/postgresql/concepts/constraints/constraints.md`.
 
 The entry should include:
 
 - An introductory paragraph that describes the concept
 - A ‘Syntax’ section that includes the syntax for the concept (if applicable)
 - An ‘Example’ section that shows the concept in use
 
 Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entrty for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by LGVjr2**
### Description

- Added the PostgreSQL Constraints.md
- described the main concept
- can you pls add the Tag 'Constraints'

### Issue Solved

-Closes #4646 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/postgresql/concepts/constraints/constraints.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'Constraints'
+  - 'Database'
+  - 'PostgreSQL'
+CatalogContent:
+  - 'getting-started-off-platform-for-data-science'
+  - 'paths/design-databases-with-postgresql'
+  - 'paths/computer-science'
```
Comment (SaviDahegaonkar) [inline content/postgresql/concepts/constraints/constraints.md:15]:
```suggestion
CatalogContent:
  - 'getting-started-off-platform-for-data-science'
  - 'paths/design-databases-with-postgresql'
  - 'paths/computer-science'
```
The paths which are present here in the CatalogContent section must be taken  from the Catalog-content.md file

File: content/postgresql/concepts/constraints/constraints.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+## Check Constraints
+- A check constraint is a condition that must be fulfilled in all rows of the table.
+```sql
+--Example:
+CREATE TABLE students (
+  student_id SERIAL PRIMARY KEY,
+  age INT,
+  --Is age between 18 and 45?
+  CONSTRAINT age_test CHECK (age BETWEEN 18 AND 45)
+  );
```
Comment (SaviDahegaonkar) [inline content/postgresql/concepts/constraints/constraints.md:27]: CREATE TABLE students (
  student_id SERIAL PRIMARY KEY,
  age INT,
  --Is age between 18 and 45?
  CONSTRAINT age_test CHECK (age BETWEEN 18 AND 45)
  );

**Review by SaviDahegaonkar — CHANGES_REQUESTED** [review]
Hello Author, 
I made some changes in this file and suggested some changes, please have a look at [Codecademy docs](https://github.com/Codecademy/docs). Looking forward for your response.

Thanks

**Comment by LGVjr2** [comment]
@SaviDahegaonkar changes done. Pls review again.

**Review by SaviDahegaonkar — APPROVED** [review]

**Review by Sriparno08 — APPROVED** [review]
Looks good, @LGVjr2! 👍

**Pull request merged**